feature_engine/data/cleaning_scripts/csopII_and_dat_cleaning.ipynb

```diff
@@ -170,22 +172,22 @@
        "</div>"
       ],
       "text/plain": [
-       "                  id  batch_num vis_img int_verb ort_img rep_man soc_pers  \\\n",
-       "0  rHPaiuXkM3Ss4rEsW          1    High     High    High    High     High   \n",
-       "1  dArSAcrzmb9bR6Pug          1    High     High    High    High     High   \n",
-       "2  sbtKHuLChG8ge734n          1    High     High    High    High      Low   \n",
-       "3  guq9kzMuDCpB4hRhK          1   Mixed     High    High    High      Low   \n",
-       "4  hSNqR8mxGD2jv3WeZ          1    High      Low    High    High     High   \n",
+       "                       id  batch_num vis_img int_verb ort_img rep_man   \n",
```

Comment:
And as a result of that bug, all the CSOP II conversation_nums have changed. I highly recommend re-running your code with this latest!